device/vr/public/cpp/features.cc

```diff
@@ -11,7 +11,7 @@ namespace device::features {
 // Enables access to articulated hand tracking sensor input.
 BASE_FEATURE(kWebXrHandInput,
              "WebXRHandInput",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
Do we really need this? Can't we use `--enable-webxr-hand-input` (or whatever is called) from Wolvic to enable it?